wallet-sdk/src/main/kotlin/org/stellar/walletsdk/anchor/AnchorTransaction.kt

```diff
@@ -30,8 +30,7 @@ sealed interface ProcessingAnchorTransaction : AnchorTransaction {
   val amountIn: String
   val amountOutAsset: String?
   val amountOut: String
-  val amountFeeAsset: String?
```

Comment:
We should keep amountFeeAsset for backward compatibility with older anchors.
We can make amountFee to be optional starting with the new version though

Reply:
The base branch of this PR should be `feature/v3`